packages/page-accounts/src/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,606 @@
+            <div className='ui--AddressMenu-section'>
+              <div className='ui--AddressMenu-sectionHeader'>
+                <div>
+                  <Icon name='tag' />
+                  {' '}
+                  {t('flags')}
+                </div>
+              </div>
+              <div className='ui--AddressMenu-flags'>
+                {isExternal && (
+                  <Label
+                    color='grey'
+                    size='tiny'
+                    tag
+                  >
+                    {t('Injected')}
+                  </Label>
+                )}
+                {isDevelopment && (
+                  <Label
+                    color='grey'
+                    size='tiny'
+                    tag
+                  >
+                    {t('Test account')}
+                  </Label>
+                )}
+                {isCouncil && (
+                  <Label
+                    color='blue'
+                    size='tiny'
+                    tag
+                  >
+                    {t('Council')}
+                  </Label>
+                )}
+                {isSociety && (
+                  <Label
+                    color='green'
+                    size='tiny'
+                    tag
+                  >
+                    {t('Society')}
+                  </Label>
+                )}
+                {isTechCommittee && (
+                  <Label
+                    color='orange'
+                    size='tiny'
+                    tag
+                  >
+                    {t('Technical committee')}
+                  </Label>
+                )}
+                {isSudo && (
+                  <Label
+                    color='pink'
+                    size='tiny'
+                    tag
+                  >
+                    {t('Sudo key')}
+                  </Label>
+                )}
+              </div>
+            </div>
+          )}
+          <div className='ui--AddressMenu-section'>
+            <div className='ui--AddressMenu-sectionHeader'>
+              <div>
+                <Icon name='share square' />
+                {' '}
+                {t('actions')}
+              </div>
+            </div>
+            {!!address && (
+              <div className='ui--AddressMenu-actions'>
+                <ul>
+                  <li>
+                    <IconLink
+                      icon='send'
+                      label={t('Transfer funds')}
+                      onClick={_toggleTransfer}
+                    />
+                  </li>
+                  {identity?.isExistent && (
+                    <li>
+                      <IconLink
+                        icon='address card'
+                        label={t('Add identity judgment')}
+                        onClick={_toggleJudgement}
+                      />
+                    </li>
+                  )}
+                  <li>
+                    <IconLink
```

Comment:
We do have a `LinkExternal` component - that caters for all (Subscan, Polkadscan)

Reply:
Seems to be just an icon with tooltip though unless I'm mistaken?

Reply:
No, it is either an icon on a text link - https://github.com/polkadot-js/apps/blob/master/packages/react-components/src/LinkExternal.tsx#L48

(Would love to use the named version in tables, but it is just unwieldily, think it is `withShort` everywhere now, except in the explorer - just because of spacing)

Reply:
![image](https://user-images.githubusercontent.com/1424473/79951533-267f2f00-8479-11ea-9f8f-9695661d2d12.png)

^^^ inside the explore block view